setup.py

```diff
@@ -27,20 +27,20 @@
 history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='django-oidc',
+    name='django-oidc-tf',
     version=version,
     description="""A Django OpenID Connect (OIDC) authentication backend""",
     long_description=readme + '\n\n' + history,
-    author='Marc-Antoine Gouillart',
-    author_email='marsu_pilami@msn.com',
-    url='https://github.com/marcanpilami/django-oidc',
+    author='Thomas Frössman',
+    author_email='thomasf@jossystem.se',
+    url='https://github.com/py-pa/django-oidc',
     packages=[
         'djangooidc',
     ],
     include_package_data=True,
     install_requires=[
-        'django>=1.8',
-        'oic>=0.7.6',
+        'django>=1.10',
```

Comment:
This works with earlier versions of Django. We are running Django 1.9.1 and this update messes with the pip install for us.
Any particular reason for the version upgrades?

Reply:
How can I redirect to my keycloak page inside a my custom decorator?